bouncingbullet/assets/js/game/draw.js

```diff
@@ -0,0 +1,50 @@
+let canvas = document.getElementById("game");
+let ctx = canvas.getContext("2d");
+
```

Comment:
https://github.com/tc39/proposal-pipeline-operator

if only...

Reply:
the worst part in this case is that it's not even really helped by chaining function calls in this specific function...

parseInt((color.substring), 16) ^ 0xFFFFFF 

is even worse

Reply:
I meant the function that follows...